trunk/SUAVE/Methods/Propulsion/Rotor_Wake/Fidelity_One/generate_fidelity_one_wake_shape.py

```diff
@@ -144,9 +143,12 @@ def generate_fidelity_one_wake_shape(wake,rotor):
     
 
     # extract airfoil trailing edge coordinates for initial location of vortex wake
-    a_sec        = rotor.airfoil_geometry   
+    if rotor.airfoil_data == None:
```

Comment:
Do this in the analysis instead for the method

Reply:
Updated